tips_and_tricks/tips_and_tricks.md

```diff
@@ -82,4 +82,4 @@ then click on changes and drag the sliders to see the document in a given time i
 
 ![TT sliders](./assets/time_travel_sliders.png)
 
-If you need to revert the document to a previous state, just drag the sliders to the desired interval and click on **Revert live version to this**, this will revert the document to that specific version and discard any modifications made afterwards.
+If you need to revert the document to a previous state, drag the slider to the desired revision and click on **Revert live version to this**. Doing so reverts the document contents to that specific version. If you have checked the **Changes** box to compare two revisions and click on **Revert live version to this**, contents are reverted to the latter of the two revisions being compared.
```

Comment:
Note that reverting a file simply create a new version of the file equal to the old file at that point in time; in particular, no work is lost!